packages/react/src/Spinner/Spinner.tsx

```diff
@@ -38,6 +45,7 @@ function Spinner({size: sizeKey = 'medium', srText = 'Loading', 'aria-label': ar
         aria-hidden
         aria-label={ariaLabel ?? undefined}
         aria-labelledby={hasHiddenLabel ? labelId : undefined}
+        className={className}
```

Comment:
This was the only component I added `className` support to, the rest already had it